feat: create client.properties for use with kafka scripts (#898)
stackabletech/kafka-operator: docs/modules/kafka/pages/getting_started/first_steps.adoc

Labels:                   app.kubernetes.io/component=broker
                          app.kubernetes.io/instance=simple-kafka-broker-default-bootstrap
                          app.kubernetes.io/managed-by=listeners.stackable.tech_listener
                          app.kubernetes.io/name=listener
                          app.kubernetes.io/role-group=default
                          app.kubernetes.io/version=3.9.1-stackable0.0.0-dev
                          stackable.tech/vendor=Stackable
Annotations:              <none>
Selector:                 listener.stackable.tech/mnt.9555cbb6f38d4b0ca1771e6d83d28e27=simple-kafka-broker-default-bootstrap
Type:                     NodePort
IP Family Policy:         SingleStack
IP Families:              IPv4
IP:                       10.105.88.52
IPs:                      10.105.88.52
Port:                     kafka  9092/TCP
TargetPort:               9092/TCP
NodePort:                 kafka  32608/TCP
Endpoints:                10.244.4.22:9092,10.244.4.24:9092,10.244.4.23:9092
Session Affinity:         None
External Traffic Policy:  Local
Internal Traffic Policy:  Cluster
Events:                   <none>
----

> In the output we see that there are three endpoints serviced here.

Reviewer: ```suggestion
The output shows that there are three endpoints serviced here.
```
